auth.go

```diff
@@ -996,7 +1004,54 @@ func (x *Central) GetAuthenticatorIdentities(getIdentitiesFlag GetIdentitiesFlag
 func (x *Central) GetRoleGroupDB() RoleGroupDB {
 	return x.roleGroupDB
 }
+func RemoveGroupFromAllUsers(c *Central, groupIDString string) error {
+	gID, err := strconv.ParseUint(groupIDString, 10, 32)
+	if err != nil {
+		return err
+	}
+	groupID := GroupIDU32(gID)
+
+	users, err := c.GetAuthenticatorIdentities(GetIdentitiesFlagDeleted)
+	if err != nil {
+		return err
+	}
+
+	for _, user := range users {
+		perm, err := c.GetPermit(user.UserId)
+		if err != nil {
+			continue
+		}
 
+		groups, err := DecodePermit(perm.Roles)
+		if err != nil {
+			return err
+		}
+
+		// Remove the group from the list of groups
+		modified := false
+		for i, g := range groups {
+			if g == groupID {
+				groups = append(groups[:i], groups[i+1:]...)
+				modified = true
+			}
+		}
+
+		// Encode the permit
+		if modified {
+			perm.Roles = EncodePermit(groups)
+			fmt.Printf("Removing group %v: setting permit for %v\n", groupID, user.Email)
+
+			//Set the permit
+			err = c.SetPermit(user.UserId, perm)
+			if err != nil {
+				fmt.Printf("Error: %v\n", err)
```

Comment:
I think we should remove this line, as the caller of this function can log this if it feels like it.